src/components/structures/RoomSubList.js

```diff
@@ -409,6 +412,9 @@ var RoomSubList = React.createClass({
         var badge;
         if (subListNotifCount > 0) {
             badge = <div className={badgeClasses}>{ FormattingUtils.formatCount(subListNotifCount) }</div>;
+        } else if (this.props.isInvite) {
+            // no notifications but highlight anyway because this is an invite badge
+            badge = <div className={badgeClasses}>!</div>;
```

Comment:
Perhaps show the number of invites, i.e. the number of items in the sublist. I'd suggest just using the `if`, passing a generic `count` to `.formatCount` above, and then deciding whether to set `count` to `subListNotifCount` or `roomCount` based on `isInvite`.

Reply:
Never mind, the "!" is by design and the number of invites is already displayed in the sublist header.

Reply:
https://github.com/vector-im/riot-web/issues/3707#issuecomment-296514435 for history sake